server/api/trend.go

```diff
@@ -145,9 +145,9 @@ func (h *trendHandler) getStoreFlow(stats statistics.StoreHotRegionsStat, storeI
 		return
 	}
 	if stat, ok := stats[storeID]; ok {
-		storeFlow = stat.TotalBytesRate
+		storeFlow = uint64(stat.TotalBytesRate)
```

Comment:
How about `storeFlow float64, regionFlows []float64` for return?

Reply:
This function is only used by `/trend` API, and the API need uint64.

Reply:
Fixed.